src/components/Panel/Panel.tsx

```diff
@@ -7,7 +7,9 @@ import Button from "../Button";
 import Card from "../Card";
 import IconButton from "../IconButton";
 import TicketDraggableContext from "./TicketDraggableContext";
-import Ticket from "../Ticket";
+import TicketComp from "../Ticket";
```

Comment:
This abbreviation is a bit unclear IMO. Just Calling it `Ticket` suffices original purpose, then you can rename Ticket to be something like `TicketDefinition`